.pre-commit-config.yaml

```diff
@@ -1,14 +1,17 @@
--   repo: git://github.com/pre-commit/mirrors-yapf
-    sha: v0.16.2
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 51b3b2624d4b1e4608eb9addd7d292cc5174a047
     hooks:
-    -   id: yapf
+    - id: black
+      args: [--line-length=88, --safe]
+      python-version: python3.6
```

Comment:
note you'll want `python_version` -- I updated the docs in https://github.com/ambv/black/pull/106 but I was not quick enough!